CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## CHANGELOG
 
+* Feature: partial helpers can access `local_assigns` values
+
+  ```html+erb
+  <%# app/views/articles/show.html.erb %>
+  <% render "section", id: "an_article" do |section| %>
+    <%= tag.h1 "An Article", id: section.labelledby %>
+  <% end %>
+
+  <%# app/views/application/_section.html.erb %>
+  <%
+    partial.helpers do
+      def labelledby
+        local_assigns.fetch(:id, "a_default_value") + "_label"
+      end
+    end
+  %>
+
+  <%= tag.section partial.yield, id:, aria: {labelledby:} %>
```

Comment:
Where does the labelledby get filled in here? Is it a local variable or does it call the custom helper?

Reply:
Ah, great catch. I was trying to use the Ruby 3.2 short-hand notation, but it needs to be called as `partial.labelledby`, since `self != partial`.